src/customHooks/useSortTable.ts

```diff
@@ -19,18 +19,36 @@ const useSortTable = (initialOrderBy: any) => {
         handleRequestSort(event, property);
     };
 
+    const sort = <T>(order: Order, first: T, second: T) => {
+        if (order == "desc")
+            return first < second
+                ? 1 : -1;
+        else
+            return first > second
+                ? 1 : -1;
+    }
+
+    const removeSymbols = (sentence: string): string => {
+        const regex: RegExp = /\$|\%|\-/g;
+        return sentence.replace(regex, '');
+    }
+
     const stableSort = <T>(array: T[], order: Order, orderBy: keyof T) => {
         return array.sort((a: T, b: T) => {
-            if (order == "desc")
-                return a[orderBy] < b[orderBy]
-                    ? 1 : -1;
-            else
-                return a[orderBy] > b[orderBy]
-                    ? 1 : -1;
-
+            let firstValue = a[orderBy];
+            let secondValue = b[orderBy];
+            if (typeof firstValue === "string" && typeof secondValue === "string") {
```

Comment:
good , but i think it might be better if we can send another parameter to indicate that it is a numeric sort 
like     const stableSort = <T>(array: T[], order: Order, orderBy: keyof T, fieldType"string"|"number"="string") => {
if(fieldType === "number") ....